### DISCUSSION TITLE
多个openai的key和多个代理地址

### DISCUSSION THREAD

**Discussion by yonmnm** [lobehub/lobehub#1549] [General]
如何配置多个openAI的key和代理地址，实现特定模型使用指定的代理和key。比如代理1使用gpt3.5模型，代理2使用4.0模型。

**Comment by yonmnm** [comment]
或者在环境配置文件里面增加一个配置，指定某些模型走固定的代理和key，其余的走全局设置。


议题#1534
![311656222-72cdc146-bf3d-49e4-a708-c38e2deca9b1.png](https://github.com/lobehub/lobe-chat/assets/88870346/e4b892b2-fd4a-423e-971a-034d198cd499)

类似这个需求，但能写到配置文件就好了，能自定义模型展示的名字

**Comment by MrChihiro** [comment]
这个可以使用oneapi实现多个openAI的key和代理地址。

**Reply by tianlichunhong** [reply]
可以介绍一下具体情况吗？如何使用？

**Reply by ITOTI-Y** [reply]
gihub有one-api的仓库，可以实现多个key的轮询

**Comment by arvinxx** [comment]
2025.02.18 callback: 新 ai provider 已经支持
